frontend/src/components/Main.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from "react";
+import { useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import garden from "../assets/garden.jpg"
+
+const Main = () => {
+    const accessToken = useSelector((store) => store.user.accessToken)
+    const navigate = useNavigate();
+
+    const logOutButton = () => {
+        window.location.reload()
+    }
+
+    useEffect(() => {
+        if (!accessToken) {
+            navigate("/login");
+        }
+    }, [accessToken]);
+
+    return (
+        <>
+            <section>
+                <div className="form-container">
+                    <h1>Your secret garden</h1>
+                    <div>
+                        <img src={garden} alt="garden" />
```

Comment:
We love the provided secret! The netlify url name made us curious of this secret garden theme :-)